UnitTests/expected/Strings-File-Structured-Only.swift.out

```diff
@@ -27,6 +27,8 @@ enum L10n {
       case HeaderTitle
       /// Here you can change some user profile settings.
       case FooterText
+      /// Welcome %@! You have %d points on your account.
+      case Points(String, Int)
```

Comment:
I was planning to do a release as soon as your PR + #131 were merged, which should be tonight 🎉